### DISCUSSION TITLE
Support for 3 wire control panel implementations - Cosy Spa

### DISCUSSION THREAD

**Discussion by pauldtc** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#549] [Ideas]
Hello,

@visualapproach thank you for your great work first of all but do you plan to also support 3 wire implementations like in the Cosy Spa / Wave Spa pumps ?

I know there already is a development by ximon in this regard but it looks rather discouraging for a DYIer like myself with very limited knowledge in electronics.

Thanks !

**Comment by visualapproach** [comment]
Hi, thanks for asking. I looked at that project a while ago. Can't remember why I chose to not implement it. Maybe not enough info to do it and no one requested it. I may take another look at it when I have time.

**Reply by pauldtc** [reply]
@visualapproach thank you for your reply, if there is any way i can help to include this in your build please let me know.

Thanks !

**Reply by visualapproach** [reply]
Sure, thanks! It will take a while before I get time to do anything with this though so don't expect anything soon
